packages/Code/src/Code.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import { typeIs } from '@leafygreen-ui/lib';
+import Code from './Code';
+
+afterAll(cleanup);
+
+const codeSnippet = 'const greeting = "Hello, world!";';
+const className = 'test-class';
+
+describe('packages/Syntax', () => {
+  describe('when multiline is true', () => {
+    const {
+      container: { firstChild: codeRoot },
+    } = render(<Code className={className}>{codeSnippet}</Code>);
+
+    if (!codeRoot || !typeIs.element(codeRoot)) {
+      throw new Error('Multiline code element not found');
+    }
+
+    test('root element renders as a <pre> tag', () => {
+      expect(codeRoot.tagName).toBe('PRE');
```

Comment:
why is this capitalized? o_O

Reply:
Because that's how `tagName` is represented in the DOM API